ivy/functional/frontends/paddle/tensor/linalg.py

```diff
@@ -151,3 +151,10 @@ def matrix_power(x, n, name=None):
 @to_ivy_arrays_and_back
 def cond(x, p=None, name=None):
     return ivy.cond(x, p=p, out=name)
+
+
+# transpose
+@with_unsupported_dtypes({"2.5.0 and below": ("uint8", "int8", "int16")}, "paddle")
```

Comment:
Hello @mohame54 @MuhammadSaeedBatikh , sorry for jumping in this PR and related issue. Actually, I ran the test cases locally for this issue of yours and I found out that the test cases are failing locally.

![image](https://github.com/unifyai/ivy/assets/86348193/2b4b18e5-5b00-41ad-a576-e220e90aea73)

I just checked the [Paddle Docs for transpose](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/transpose_en.html) and there we can see, these are the dtypes supported.

![image](https://github.com/unifyai/ivy/assets/86348193/383eca3c-283a-4cdb-b53d-c6eb16c16114)

So I tried using supported dtypes with the above mentioned dtypes, while defining the transpose function, and all the test cases passed successfully !

![image](https://github.com/unifyai/ivy/assets/86348193/32b48faf-4196-4391-8033-98a9d47cb317)

Thank you.